### PR TITLE
Use ubuntu-focal-20.04 by default on AWS

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
Use ubuntu-focal-20.04 by default on AWS

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Use ubuntu-focal-20.04 by default on AWS
```

### CODE REVIEW

File: pkg/scripts/os.go [hunk 1]
```diff
@@ -57,9 +56,9 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get install --option "Dpkg::Options::=--
 
 {{- if .CONFIGURE_REPOSITORIES }}
 curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-curl -fsSL https://download.docker.com/linux/${ID}/gpg | sudo apt-key add -
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 
-echo "deb https://download.docker.com/linux/${ID} $(lsb_release -sc) stable" |
+echo "deb https://download.docker.com/linux/ubuntu bionic stable" |
```
Comment (xmudrii) [inline pkg/scripts/os.go:61]: Can we add a TODO to replace this once 1.16 is not supported anymore?

Comment (kron4eg) [reply pkg/scripts/os.go:62]: 👍

**Pull request merged**